src/test/java/ParserTest.java

```diff
@@ -0,0 +1,61 @@
+package simpli.parser;  //same package as the class being tested
+
+import org.junit.jupiter.api.Test;
+import simpli.exceptions.ActionException;
+import simpli.exceptions.TaskException;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Should these import statements be organised differently? Could we also consider avoiding the use of wildcard imports?